src/test/java/com/epam/tc/hw9/PositiveApiTestsRestTrelloService.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.tc.hw9;
+
+import com.epam.api.dto.BoardDto;
+import com.epam.api.dto.CardDto;
+import com.epam.api.dto.ListDto;
+import io.restassured.response.Response;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.Matchers.*;
+
+public class PositiveApiTestsRestTrelloService extends BaseTest {
+    @DataProvider
+    public Object[][] boardData() {
+        return new Object[][]{
+                {"FirstBoard"},
+                {"SecondBoard"},
+                {"ThirdBoard"},
+        };
+    }
+
+    @Test(description = "A test that verifies the correctness of creating a board",
+            dataProvider = "boardData")
+    public void createBoardTest(String nameBoard) {
+        boardDto.setName(nameBoard);
+        restTrelloService.createNewBoard(boardDto);
+        boards = restTrelloService.getAllBoards();
+        checkCountBoards(boards.length, defCountBoard);
```

Comment:
here is better to check board creation response

like 
BoadDto newBoard = boardDto.setName(nameBoard);
newBoard = restTrelloService.createNewBoard(newBoard);

boards = restTrelloService.getAllBoards();
checkBoardsIsInAllBoards(newBoard, boards);

Reply:
+